outofcore/src/cJSON.cpp

```diff
@@ -80,7 +81,15 @@ void cJSON_InitHooks(cJSON_Hooks* hooks)
 static cJSON *cJSON_New_Item()
 {
 	cJSON* node = static_cast<cJSON*> (cJSON_malloc(sizeof(cJSON)));
-	if (node) memset(node,0,sizeof(cJSON));
+	if (node) {
```

Comment:
Why not something similar to `if (node) { *node = cJSON{}; }`

Reply:
Cool, it worked